### ISSUE TITLE
Class Illuminate\Database\Eloquent\Factory does not exist

### ISSUE THREAD

**Issue by riasvdv** [orchestral/testbench-core#44]
- Testbench Version: 6.0-dev
- Laravel Version: 8.0-dev
- PHP Version: 7.4
- Database Driver & Version: MySQL 5.7

### Description:

The `WithFactories` concern throws this exception:

```
ReflectionException: Class Illuminate\Database\Eloquent\Factory does not exist

/Users/rias/Code/laravel-mailcoach/vendor/laravel/framework/src/Illuminate/Container/Container.php:809
/Users/rias/Code/laravel-mailcoach/vendor/laravel/framework/src/Illuminate/Container/Container.php:691
/Users/rias/Code/laravel-mailcoach/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:796
/Users/rias/Code/laravel-mailcoach/vendor/laravel/framework/src/Illuminate/Container/Container.php:637
/Users/rias/Code/laravel-mailcoach/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:781
/Users/rias/Code/laravel-mailcoach/vendor/orchestra/testbench-core/src/Concerns/WithFactories.php:31
/Users/rias/Code/laravel-mailcoach/vendor/orchestra/testbench-core/src/Concerns/WithFactories.php:18
/Users/rias/Code/laravel-mailcoach/tests/TestCase.php:30
/Users/rias/Code/laravel-mailcoach/tests/Traits/UsesMailcoachModelsTest.php:19
```

When testing with Laravel 8 and Testbench 6.0-dev

Seems like this class is moved in Laravel 8: https://github.com/laravel/framework/tree/master/src/Illuminate/Database/Eloquent

**Comment by crynobone** [comment]
https://github.com/orchestral/testbench/tree/master#class-illuminatedatabaseeloquentfactory-not-found

**Comment by driesvints** [comment]
Think what rias was trying to say is that the upcoming 6.0 release of testbench is incompatible with Laravel 8 atm. I think `WithFactories.php` should definitely be patched since 6.0 is specifically meant for Laravel 8. We can't update Passport atm because of this. We don't want to require legacy factories in Passport for its next release. We're migrating to the new ones.
